### ISSUE TITLE
compiling tests on windows fails

### ISSUE THREAD

**Issue by meshula** [AcademySoftwareFoundation/openexr#934]
testCompression.cpp needs #include <algorithm> to resolve the std::min macro.

(I'm still waiting on EasyCLA)

**Comment by cary-ilm** [comment]
Can you point to the log for the failed build? I'm happy to fix it but I'm not seeing it, and I also don't see where std::min() is called in testCompression.cpp.

**Comment by meshula** [comment]
I wrote that too late at night, should have dropped the messages in. Also, it's max, not min... Also I see the markdown ate my message.

```cpp
#include <algorithm>
```
is what that note was supposed to read.

```
C:\Projects\openexr\src\test\OpenEXRTest\testCompression.cpp(470,59): error C3861: 'max': identifier not found [C:\Projects\openexr\build\src\test\OpenEXRTest\OpenEXRTest.vcxproj]
C:\Projects\openexr\src\test\OpenEXRTest\testCompression.cpp(470,49): error C3861: 'max': identifier not found [C:\Projects\openexr\build\src\test\OpenEXRTest\OpenEXRTest.vcxproj]
```

**Comment by cary-ilm** [comment]
#937 should hopefully fix it. It's std::max is in <limits>, right? 

Can do you know why you're getting this error when our Windows CI build succeeds?

**Comment by meshula** [comment]
I tested 937 just now. limits is insufficient, I had to add algorithm.

I haven't had time to dig into why the CI succeeds. I thought I would look into the other issue where Windows build failures were reported, and discovered that I saw a few blocking issues locally, which I've reported. My best theory is that I have a more recent Visual Studio than the CI runs.

There could be further issues, I won't know until the Imath install prefix error is resolved.
